### ISSUE TITLE
Change Language of the UI

### ISSUE THREAD

**Issue by creeper82** [futo-org/android-keyboard#755]
### Use Cases

The UI (settings, etc.) defaults to system, but in my case it is barely translated (polish), causing a mix of untranslated and translated menus

### Proposal

A setting to change the language, or simply to force english, which should be enough for most users

**Comment by dean-futo** [comment]
@creeper82 will you please elaborate on what you're requesting?

**Comment by wertzuz** [comment]
Ability to manually change app language of FUTO keyboard instead of it being the system-language.

**Comment by creeper82** [comment]
Yes. Language of the application, and not language in context of layouts or spellchecking.

**Comment by dean-futo** [comment]
Got it. Can do.

**Comment by abb128** [comment]
In [Nightly](https://docs.keyboard.futo.org/improvements/nightly) translation has been added for Polish and some other common languages. I'm not 100% certain about the accuracy but we'll have a way for users to improve translations
